caesars-palace/blocks/slider/slider.js

```diff
@@ -24,6 +19,51 @@ const setSliderPosition = (currentTranslate, slider) => {
   slider.style.transform = `translateX(${currentTranslate}px)`;
 };
 
+/**
+ * Build the preview of a text with ellipsis
+ * @param {String} text Text that will be shortened
+ * @param {Integer} width Width of container
+ * @param {Integer} maxVisibleLines Max visible lines allowed
+ * @param {*} suffix Suffix to use for ellipsis
+ *  (will make sure text+ellipsis fit in `maxVisibleLines`)
+ * @param {*} options Text styling option
+ *
+ * @return The ellipsed text (without ellipsis suffix)
+ */
+function buildEllipsis(text, width, maxVisibleLines, suffix, options = {}) {
```

Comment:
Is this the same function that Carousel uses? Could it be moved to `scripts.js` as a shared function?

Reply:
I think it could. I was wondering where to put it, because it's relatively specific. But if you tell me I can add it there, I will!

Reply:
Moved it accordingly in https://github.com/hlxsites/caesars/pull/112, will refactor here too once https://github.com/hlxsites/caesars/pull/112 is merged.